src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It's good that you've specified a custom font for your page. However, it's also a good idea to specify a fallback font, in case the custom font fails to load. Fallback fonts are usually generic font families that are available on all systems. For example, you can use a generic font like 'sans-serif' as a fallback. Here's how you can do it: `font-family: Roboto, sans-serif;`